config/base/looker.yml.tmpl

```diff
@@ -0,0 +1,7 @@
+looker_repository: git@gitlab.com:getindata/dataops/looker/pipeline-example-looker.git
+looker_repository_username: "[[ pipeline_owner ]]"
+looker_repository_email: "[[ pipeline_owner ]]@getindata.com"
+looker_project_id: [[ project_name ]]
+looker_webhook_secret: provide_lookers_webhook_secret
```

Comment:
I think that is a security violation. We will need to use it from envs during CICD to not commit into git repo.